Explorer/Assets/DCL/AvatarRendering/Wearables/Equipped/IEquippedBodyShape.cs

```diff
@@ -0,0 +1,11 @@
+using DCL.AvatarRendering.Wearables.Components;
+
+namespace DCL.AvatarRendering.Wearables.Equipped
+{
+    public interface IEquippedBodyShape
```

Comment:
I dont really like this approach. The more things we will have to update in the profile, the more classes we will have to create. Feels like over-engineer design